lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -398,6 +398,8 @@ def _call(self, method: Callable, params: ApiParams, context: ApiContext):
                 es_request.get("kwargs").get("json"),
             )
         except Exception as e:
+            if isinstance(e, APIAbort):
+                raise
             raise APIInternalError("Elasticsearch assembly error") from e
```

Comment:
Wouldn't this be better expressed as follows?
```python
        except APIAbort:
                raise
        except Exception as e:
            raise APIInternalError("Elasticsearch assembly error") from e
```
(Same number of lines....  🙂)

Reply:
I originally coded it that way but decided I liked this way better.

Reply:
🦐